C-Plus-Plus/sort/counting_sort.cpp

```diff
@@ -0,0 +1,72 @@
+//
+// Created by Sarthak-9 on 30-12-2020.
+// Count Sort Algorithm for Sorting an array of small integers quickly
+//
+// Worst complexity: O(n+k)
+// Average complexity: O(n+k)
+// Space complexity: O(n+k)
+// where n is the number of elements in input array and k is the range of input.
+//
+
+
+#include <iostream>
+
+using namespace std;
+
+int Max(long long int A[], long long int n){
+    long long int max = -32768;
```

Comment:
Its better to assign the 1st element of the array (A[0]) as max initially while checking for maximum element in an array.